import_export/templates/admin/import_export/resource_fields_list.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}
+{% block fields_help %}
+  <p>
+    {% trans "This exporter will export the following fields: " %}
+    {% if fields_list|length <= 1 %}
```

Comment:
Only lines 5-14 can be included in `resource_fields_list.html`, because of the line:

> This exporter will export the following fields

If used in a shared template file, this text will be incorrect when importing fields:

![imp](https://github.com/django-import-export/django-import-export/assets/6249838/5a275ae6-edea-4ea2-8900-58f1feeffd65)

Therefore the text on line 4 will have to be retained in export.html and import.html

Reply:
right:)